data/noiseprofiles.schema

```diff
@@ -18,9 +18,9 @@
         "properties": {
           "maker": {
             "type": "string",
-            "pattern": "^[a-zA-Z]+$",
+            "pattern": "^[a-zA-Z ]+$",
```

Comment:
i added a space here to match YI M1 maker name. Please review changes in *.schema